pkgs/tools/text/gucci/default.nix

```
@@ -1,29 +1,31 @@
-{ lib, buildGoPackage, fetchFromGitHub }:
+{ lib, buildGoModule, fetchFromGitHub, testers, gucci }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "gucci";
-  version = "0.1.0";
-
-  goPackagePath = "github.com/noqcks/gucci";
+  version = "1.5.4";
 
   src = fetchFromGitHub {
     owner = "noqcks";
     repo = "gucci";
-    rev = version;
-    sha256 = "0ksrmzb3iggc7gm51fl0jbb15d0gmpclslpkq2sl2xjzk29pkllq";
+    rev = "1c7800b2079702ff99dcc0d9bc2934d797c94812";
```

If upstream tags version, it will be nice not to hardcode rev so we can get auto updates from r-ryantm.

```suggestion
    rev = version;
```

---

There is a `1.5.4` branch also. `1.5.4` is ambiguous.

---

I guess we can use `rev = "refs/tags/${version}";` then :thinking: